src/index.js

```diff
@@ -107,7 +107,13 @@ class ProfilerPlugin {
     }
   }
 
-  getFileUploadMeta() {
+  async getFileUploadMeta() {
```

Comment:
Would this be better as a getter?

Reply:
It looks as though this is following/importing the interface to this functionality in core. Here, it would make more sense as a getter, but I also don't think there's a lot of benefit to changing this interface.